sklearn/metrics/_plot/base.py

```diff
@@ -82,9 +82,7 @@ def _get_response(X, estimator, response_method, pos_label=None):
         The class considered as the positive class when computing
         the metrics.
     """
-    classification_error = "Expected a binary classifier, but got {}".format(
-        estimator.__class__.__name__
-    )
+    classification_error = f"Expected 'estimator' to be a binary classifier, but got {estimator.__class__.__name__}"
```

Comment:
This line is surely too long and will be caught by triggering an error on the CI

Reply:
I split it and added a new line,   hope my line feed method meets the code specification.